File: ht13-spring-security-authorization/ht13-server/src/main/java/ru/otus/kulygin/dto/list/GenreListDto.java
package ru.otus.kulygin.dto.list;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import ru.otus.kulygin.dto.GenreDto;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class GenreListDto {

    private List<GenreDto> genres;

}

File: ht17-docker/ht17-client/src/main/web/src/app/models/list/genre-list-dto.ts
import {GenreDto} from "../genre-dto";

export class GenreListDto {

    private _genres: GenreDto[] = [];

    constructor(genres: GenreDto[]) {
        this._genres = genres;
    }

    get genres(): GenreDto[] {
        return this._genres;
    }

    set genres(value: GenreDto[]) {
        this._genres = value;
    }

}